src/Logary/Targets_Graphite.fs

```diff
@@ -71,7 +71,7 @@ module Impl =
   /// All graphite messages are of the following form.
   /// metric_path value timestamp\n
   let createMsg (path : String) value (timestamp : Instant) =
-    let line = String.Format("{0} {1} {2}\n", path, value, timestamp.Ticks / NodaConstants.TicksPerSecond)
+    let line = String.Format("{0} {1} {2}\n", path, value, (timestamp |> Internals.Date.getTicks ) / NodaConstants.TicksPerSecond)
```

Comment:
Perhaps extension method on `Instant` like `instant.ticks`? Why shim it in the first place? Can't NodaTime be upgraded instead?